src/Domain/CoordinateMaps/TimeDependent/Rotation.cpp

```diff
@@ -174,6 +150,11 @@ void Rotation<Dim>::pup(PUP::er& p) {
   if (version >= 0) {
     p | f_of_t_name_;
   }
+
+  // No need to pup this because it is uniquely determined by f_of_t_name_
+  if (p.isUnpacking()) {
+    f_of_t_names_.insert(f_of_t_name_);
```

Comment:
I don't know whether we ever unpack into something that hasn't just been default-constructed, but probably best to clear this first just in case.